extra_vars.yml

```diff
@@ -19,6 +19,12 @@ os_image_url: https://download.opensuse.org/repositories/Cloud:/Images:/Leap_15.
 os_image_checksum: sha256:https://download.opensuse.org/repositories/Cloud:/Images:/Leap_15.5/images/openSUSE-Leap-15.5.x86_64-NoCloud.qcow2.sha256
 os_image_name: openSUSE-Leap-15.5.x86_64-NoCloud.qcow2
 
+# NOTE: this should be *your* path to the OS image if you want to use an image locally.
+# For example, if you want to use SLEMicro you should download it from SCC and then
+# set this variable to the path of the downloaded image (probably to be consumed by EIB).
+# This parameter is optional unless you want to use a local image instead of downloading using the url above. If you don't define the value, it will download the image from the url above.
+os_image_path:
```

Comment:
Its fine to document this here, but IMO it would be better to avoid requiring users to modify this file, one approach would be to default to an environment variable, e.g `OS_IMAGE_PATH` - another would be to document passing `-e os_image_path=foo` to the script (since [all extra script variables are passed to ansible-playbook](https://github.com/suse-edge/metal3-demo/blob/main/02_configure_host.sh#L17))

Note that if we pick the environment variable approach we'll need to drop the `is defined` from the conditional (which is probably a good idea anyway as it'll evaluate to true for any empty string) e.g:

```
---
- name: Testing
  hosts: localhost
  connection: local
  vars:
    os_image_path: "{{ lookup('env', 'OS_IMAGE_PATH') }}"
  tasks:
    - debug:
        msg: "os_image_path defined {{ os_image_path }}"
      when: os_image_path
    - debug:
        msg: "os_image_path NOT defined"
      when: not os_image_path
```

Reply:
I agree with the env var approach. We still need the Leap for the mgmt cluster part. So I will add the env to keep the local image location: path + image name using an enviroment var. 
I will modify the EIB part with that env, keeping the image cache as is (because we need that images cached for mgmt cluster)